pagopa/io-app
# fix(IDPay): [IODPAY-212] "Tap to scroll" floating button in IDPay onboarding screen

##  ⚠️ This PR depends on https://github.com/pagopa/io-app/pull/4536

## Short description
This PR adds a "tap to scroll" button in IDPay initiative onboarding screen.

https://user-images.githubusercontent.com/6160324/232086854-6ab2c3ca-2d69-4338-9c89-c15fc0045e1f.mp4

## List of changes proposed in this pull request
- Added `ScaleInOutAnimation` custom animation 
- Added `ForceScrollDownView` which renders a scroll view with a button to scroll to the bottom
- Added skeleton to `InitiativeDetailsScreen`
- Added `ForceScrollDownView` to `InitiativeDetailsScreen`

## How to test
`yarn test:dev` and, with the `io-dev-server`, navigate to the onboarding screen of an IDPay initiative. Check that the "tap to scroll" button is working correctly.
